README.md

```diff
@@ -1,79 +1,53 @@
+    <a href="https://iota.stackexchange.com/" style="text-decoration:none;"><img src="https://img.shields.io/badge/StackExchange-9cf.svg?logo=stackexchange" alt="StackExchange"></a>
+    <a href="https://github.com/iotaledger/bee/blob/master/LICENSE" style="text-decoration:none;"><img src="https://img.shields.io/github/license/iotaledger/bee.svg" alt="Apache 2.0 license"></a>
+</p>
+      
+<p align="center">
+  <a href="#about">About</a> ◈
+  <a href="#design">Design</a> ◈
+  <a href="#supporting-the-project">Supporting the project</a> ◈
+  <a href="#joining-the-discussion">Joining the discussion</a> 
+</p>
 
-## Outline
+---
 
-+ [Motivation]
-+ [Milestones]
-+ [Contributing]
-+ [Why Rust?]
+# About
 
-## Motivation
-[Motivation]: #motivation
+The IOTA Foundation aims to allow machines of all performance levels to
+contribute to the IOTA network, from microcontrollers and single-board
+computers, to phones, web browsers, desktop machines, and servers.
 
-Our primary motivation is to make IOTA an exemplary open source project, with great documentation, clear examples of
-increasing complexity, and well-tested code. Entry into the IOTA developer ecosystem should be as straight forward as
-possible, so that people can start contributing, extending, and building their ideas on top of the IOTA network using
-the Bee framework. As such, we want to be ready for development driven by our own dev and research teams, as well as
-novel ideas coming out of our community.
+Therefore, Bee is being developed as a modular collection of extendable crates, which expose foreign function interfaces (FFIs) for the next iteration of client libraries.
 
-Bee will provide one central reference implementation of the most important data structures and algorithms, which will
-be verified and eventually certified. By this approach, we hope that improvements in any core components will quickly
-propagate to all other libraries building on them, rather than having to fix each single implementation we encounter.
+**Note:** You can find details about future development plans in our [roadmap](https://roadmap.iota.org).
 
-The IOTA Foundation aims to allow machines of all performance levels to contribute to the IOTA network, from
-microcontrollers and single-board computers, to phones, web browsers, desktop machines and servers. We therefore want to
-implement Bee's constituent libraries from the ground up, to allow the import of each library on its own, with each
-library having as few dependencies as possible. Having microcontrollers in mind, we want these libraries to not rely on
-a runtime — this includes language runtimes such as garbage collection, but also OS integration (which, in the case of
-Rust, is provided by its `libstd` standard library).
+## Design
 
-## Milestones
-[Milestones]: #milestones
+Bee will provide one central reference implementation of the most important
+data structures and algorithms, which will be verified and eventually
```

Comment:
if **by verified** you mean the RFC process, then I think it makes sense to mention it here.